VictoriaAlbert/VAMDetailViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  VAMDetailViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Karen Fuentes on 11/10/16.
+//  Copyright © 2016 Karen Fuentes. All rights reserved.
+//
+
+import UIKit
+
+class VAMDetailViewController: UIViewController {
+    var recordThatWasSelected: Record?
+    @IBOutlet weak var fullImage: UIImageView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        loadData()
+    }
+
+ 
+    func loadData() {
+        let fullImage = recordThatWasSelected!.imageID
+        let firstSixINeed = fullImage.startIndex..<fullImage.index(fullImage.startIndex, offsetBy: 6)
+        let imageSubscript = fullImage[firstSixINeed]
+        let imageURL = "http://media.vam.ac.uk/media/thira/collection_images/\(imageSubscript)/\(fullImage).jpg"
```

Comment:
It would be better to refactor this code that takes the string prefix so it's only defined in one place. Since it's an operation on the data it could be a computed property or function of your Record object.